health/service.go

```diff
@@ -95,7 +90,7 @@ func m3ctlHealthCheck(w http.ResponseWriter, r *http.Request) {
 // RegisterHandlers registers health handlers.
 func (s *service) RegisterHandlers(router *mux.Router) {
 	log := s.iOpts.Logger()
-	router.HandleFunc("", m3ctlHealthCheck)
+	router.HandleFunc(s.URLPrefix(), m3ctlHealthCheck)
```

Comment:
I am a little confused as to why the health check was at the root, and how this worked, to begin with.